_posts/2019/2019-07-28-consider-value-objects.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Consider Value Objects"
+description: "Making text readable on"
+layout: article
+category: ruby
+image:
+  base: '2019/consider-value-objects'
+  alt: 'Cubes'
+  credit: Christian Fregnan
+  source: "https://unsplash.com/photos/TAVKURx-xLw"
+
+---
+
+Often we use simple concepts in our app, that don't start out as objects.
+
+You'll often find yourself creating multiple view helpers or passing a value (or several) around several methods, or within a model.
+
+This loose concept can be refactored into a “Value Object”.
+
+
+## Instead of...
+
+...just using a helper.
```

Comment:
And you didn't refer to helpers in your intro, maybe you should.

Also, colon, not full-stop. :-D